portals/application/configs/klear/TerminalModelsList.yaml

```diff
@@ -127,7 +127,7 @@ production:
       labelOnEdit: true
       action: run-generic-code
       class: ui-silk-control-play-blue
-    runSpecificCode_dialog:
+    runSpecificCode_dialog: &runSpecificCode_dialog
```

Comment:
We usually call YAML links ending with **Link** word
Also, where is this link used :thinking: ?